.editorconfig

```diff
@@ -371,3 +371,7 @@ dotnet_diagnostic.SA1651.severity = none
 dotnet_diagnostic.SA1652.severity = none
 dotnet_diagnostic.SX1309.severity = none
 dotnet_diagnostic.SX1309S.severity = none
+
+# Source generator settings
+property_changed_raise_method_name = OnPropertyChanged,RaisePropertyChanged
```

Comment:
This probably should be moved to a `.editorconfig` file in `test/Saritasa.Tools.SourceGenerator.Benchmarks` directory?